packages/suite/src/views/wallet/staking/components/EthStakingDashboard/components/EverstakeFooter.tsx

```diff
@@ -0,0 +1,46 @@
+import styled, { useTheme } from 'styled-components';
+import { Image } from '@trezor/components';
+import { spacingsPx } from '@trezor/theme';
+import { Translation, TrezorLink } from 'src/components/suite';
+
+const Wrapper = styled.div`
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    gap: ${spacingsPx.md};
+    justify-content: space-between;
+    padding-top: ${spacingsPx.xl};
+    border-top: 1px solid ${({ theme }) => theme.borderOnElevation1};
+    margin-top: ${spacingsPx.xxl};
+`;
+
+const Left = styled.div`
+    display: flex;
+    align-items: center;
+    color: ${({ theme }) => theme.textSubdued};
+    gap: ${spacingsPx.xs};
+`;
+
+export const EverstakeFooter = () => {
+    const theme = useTheme();
+    const isDarkMode = theme.THEME === 'dark';
+
+    return (
+        <Wrapper>
+            <Left>
+                <Translation id="TR_STAKE_PROVIDED_BY" />{' '}
+                <Image
```

Comment:
How about adding just one everstake logo svg and change color using css?

Reply:
Done. https://github.com/trezor/trezor-suite/pull/11543/commits/7295f09ab08677797bacb5c857312e106242001d